src/lib/info_tree.rs

```diff
@@ -56,39 +52,92 @@ const REL_SUBTREES_SIZE: usize = 3;
 // Size of the node type vector.
 const NODE_TYPE_SIZE: usize = 3;
 
+enum PathIdx {
+    LEFT,
+    RIGHT,
+    HEAVY,
+    BOTH
+}
+
+const LEFT: usize = 0;
```

Comment:
Shouldn't we also have deleted these lines? Otherwise `PathIdx::LEFT` and `LEFT` (etc.) can get out of sync.

Reply:
Are you looking at an old commit? I can't see that left in my text editor?

Reply:
Which commit removed it?

Reply:
8d66cb57d3c794447ae4e5bd35a97130a91c2d71

Reply:
Got it.